src/main/java/org/folio/entlinks/service/reindex/AuthorityReindexJobRunner.java

```diff
@@ -110,17 +110,25 @@ private AuthorityDto toAuthority(ResultSet rs,
       authority.setSubjectHeadingCode(subjectHeadingCode != null ? subjectHeadingCode.charAt(0) : null);
 
       var array = rs.getArray(Authority.SFT_HEADINGS_COLUMN);
-      var sftHeadings = objectMapper.readValue(array.toString(), headingRefType);
-      authority.setSftHeadings(Arrays.asList(sftHeadings));
+      if (array != null) {
```

Comment:
fix for NPE which is found via integration test [org.folio.entlinks.controller.AuthorityReindexControllerIT.cancelReindexJob_positive_shouldCancelSubmittedReindexJob](https://jenkins-aws.indexdata.com/job/folio-org/job/mod-entities-links/job/master/129/testReport/org.folio.entlinks.controller/AuthorityReindexControllerIT/cancelReindexJob_positive_shouldCancelSubmittedReindexJob/)